src/core/config/config.service.ts

```diff
@@ -214,6 +217,9 @@ export class ConfigService {
       serverTimestamp: new Date().toISOString(),
       theme: this.ui.theme || 'deep-purple',
       menuMode: this.ui.menuMode || 'default',
+      terminalPersistence: Boolean(this.ui.terminalPersistence),
+      terminalShowWarning: Boolean(this.ui.terminalShowWarning),
+      terminalBufferSize: this.ui.terminalBufferSize || 50000,
```

Comment:
we have a relatively new globals file for default constants like this, i think it would good to make use of it:
- https://github.com/homebridge/homebridge-config-ui-x/blob/latest/src/globalDefaults.ts